src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+2.0.25
+++++++
+Minor fix
```

Comment:
Nit: could you just mention the timezone change here in the history

Reply:
It is not a user facing change, so we just mention "minor change"